marklogic-data-hub-central/ui/src/components/modeling/entity-type-modal/entity-type-modal.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
+import EntityTypeModal from './entity-type-modal';
+
+describe('EntityTypeModal Component', () => {
```

Comment:
Is "Add" button disabled in the absence of required field? Can add asserttion for that.
Can add a test to verify form error to test Name is a required field.
Will be good to include tests to update description(and edit entity) here, unless there is a separate story for that.

Reply:
I've added disabling of the "Add" button and updated the tests. There is a separate story for editing